reference/src/client/test/sa_key_derive_common_root_key_ladder.cpp

```diff
@@ -25,14 +25,16 @@
 
 using namespace client_test_helpers;
 
```

Comment:
Missing second asterisk removal in file header.

Reply:
Done

Reply:
Done